Add abijwe

akashbijwe opened the pull request:
All the observations from this thread are resolved, please add this readme.
https://github.com/matiassingers/awesome-readme/issues/256#event-12704335368

Robson commented:
Thank you for looking into these things - glad they were useful. 

Just reviewing the original list:

> When submitting the pull request, please use the URL: https://github.com/akashbijwe/akashbijwe#readme

✅ Done 

>  Some technology names need little tweaks:
>     Javascript -> JavaScript.
>     Typescript -> TypeScript.
>     Mongo DB -> MongoDB.
>     There may be more.

✅ Done 

> Some wording changes would be good:
> "In a free time I love to contribute on open-source community." -> "In my free time I love to contribute to the open-source community."

✅ Done 

> There's a website URL listed in an image, but no clickable link to it.

ℹ I think this would be beneficial because it'll let users access your site easier and it'll increase engagement for your site. Could be achieved by splitting the contact image into two (one for the e-mail and one for the website), then wrap the image in an _a_ tag to link to the website. Happy to add this to the repository without that though.

> Several of the social media links are 404 errors, private accounts, have no content or are closed services.

✅ Done 

> Possibly a temporary issue, but some social images at the end aren't working for me. Notably the 5th and 6th ones. They say "Not found" when going to the image locations.

✅ Done

> I like the images used throughout, but this doesn't feel accessible. Maybe there is some way to improve that, such as using alt text for the images. I'm not sure if there is a way to achieve this content using tables/etc, but possibly worth investigating.

ℹ Definitely worth considering this from an accessibility perspective. All it would take is putting the text on the alt tags, then the readme would work normally and with screenreaders. Happy to add this to the repository without that though.

meduzen commented:
Hi, shimming in! On top of the missing images alt text, I have two concerns regarding this kind of repository promoted as _awesome_:
- it’s over 500 KB of image files for what would likely have less than 80 KB (compressed) when done as a website.
- it’s not practical on mobile, where it looks like this and is a strong accessibility issue:
<img width="368" alt="The website https://github.com/akashbijwe/akashbijwe#readme viewed on a phone" src="https://github.com/matiassingers/awesome-readme/assets/9340937/6514e569-0764-4a71-b103-59e9638ed91d">

Thou it’s visually appealing, I would rather encourage to build this as a website, or at least to split every images in two (left/right) and inline them so it stacks up on mobile, like a responsive website 😃.

akashbijwe commented:
Thanks @Robson for reviewing it thoroughly 💯, I have added the alt to the images.

akashbijwe commented:
Hello @meduzen ,
Yes, you are right! I have compressed the images all image size is not less than 80kb but It's now reduced to 180kb from 500kb.

Robson commented:
Hi @akashbijwe Do you have any thoughts on the two remaining items? That's the accessibility and the unclickable website URL.

akashbijwe commented:
Hey @Robson,
1. I have added the `alt` for all the images
2. For this I think it's just fine, I have added the link to my website in the `Find me around the web 🌎` section. 
> I think this would be beneficial because it'll let users access your site easier and it'll increase engagement for your site. Could be achieved by splitting the contact image into two (one for the e-mail and one for the website), then wrap the image in an a tag to link to the website. Happy to add this to the repository without that though.

Thank you for the thoroughly reviewing my profile page, really appreciate it. 💯

Robson commented:
Hi @akashbijwe Thank you for looking into this. Visually this readme looks good, but I have too many issues with it to be able to consider it _awesome_. All issues have already been raised in this ticket, but I will repeat them in case that is helpful:

- Insufficient alt text. The alt text is just a title, so doesn't solve the accessibility issue.
- The text is very tiny on mobile.
- The format being used is a website rather than a GitHub readme.

I see three ways forward from here:

- This readme has great potential to be a mobile-compliant and accessible website.

or

- The readme could be redone to separate the text (i.e. the text shouldn't be in images) and the images. We're working with limited markdown options on GitHub, so this would need a new layout. This could solve all issues raised above. If this was my readme, this would be my preferred option because this could be turned into an aesthetically-pleasing readme, whilst solving all issues.

or

- Do nothing. In this situation the readme is just not suitable for adding to this repository.

Robson commented:
Activity on this has slowed, so I'm going to close this pull request. Do feel free to reopen, if all issues raised are resolved.
